routes/discord_genlink.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const router = express.Router();
+router.use(express.json());
+const discordService = require('../services/discord_bot_service');
+const { respondWithError } = require("./helpers");
+
+router.post('/generate-invite', async (req, res) => {
+  try {
+    if (!req.userId) {
+      return res.status(401).json({ error: 'User with provided UID not found' });
```

Comment:
the error message seems to be misleading; also, could we reuse existing middlewares instead?